src/main/scala/io/iohk/ethereum/jsonrpc/AkkaTaskOps.scala

```diff
@@ -0,0 +1,20 @@
+package io.iohk.ethereum.jsonrpc
+
+import akka.actor.{Actor, ActorRef}
+import akka.util.Timeout
+import monix.eval.Task
+
+import scala.reflect.ClassTag
+
+object AkkaTaskOps { self: ActorRef =>
```

Comment:
what is purpose of `self: ActorRef` here?

Reply:
Not needed. (Leftover after the previous solution with the trait.)